samples/client/petstore/go/go-petstore/README.md

```diff
@@ -22,7 +22,7 @@ go get golang.org/x/net/context
 Put the package under your project folder and add the following in import:
 
 ```golang
-import sw "./petstore"
+import petstore "github.com/GIT_USER_ID/GIT_REPO_ID"
```

Comment:
it would be nice to configure the sample client to generate correct import name:
```suggestion
import petstore "github.com/OpenAPITools/openapi-generator/samples/client/petstore/go/go-petstore/petstore"
```

Reply:
I would assume that GIT_USER_ID and GIT_REPO_ID are properly set in the generator - as those are the vars being used by the template. Is that not so? For generated clients that do set these values, this works as you suggested.

Reply:
yes, these are set via options (global).

Reply:
Ok, so this is correct then as it is right now?